dotcom-rendering/src/web/components/EnhancePinnedPost.importable.tsx

```diff
@@ -87,6 +94,13 @@ export const EnhancePinnedPost = () => {
 		};
 	}, []);
 
+	useEffect(() => {
+		pinnedPostButton?.addEventListener('keydown', keyListener);
+
+		return () => {
+			pinnedPostButton?.removeEventListener('keydown', keyListener);
+		};
+	}, []);
```

Comment:
We’ve got the `useOnce` pattern for this kind of thing, makes it clearer that this will only execute once.